sdk/js/src/token_bridge/redeem.ts

```diff
@@ -219,3 +235,121 @@ export async function redeemOnAlgorand(
     senderAddr
   );
 }
+
+/**
+ * This basically just submits the VAA to Near
+ * @param client
+ * @param tokenBridge Token bridge ID
+ * @param vaa The VAA to be redeemed
+ * @returns Transaction ID(s)
+ */
+export async function redeemOnNear(
+  client: nearAccount,
+  tokenBridge: string,
+  vaa: Uint8Array
+): Promise<String> {
+  let p = _parseVAAAlgorand(vaa);
```

Comment:
Is this really an Algorand specific VAA parser or is it a generic VAA parser?

Reply:
This is a generic parser but it got renamed to ...Algorand when we merged back then. .. We have multiple parsers and it would be useful to merge them